src/pages/home/HomePage.tsx

```diff
@@ -8,15 +9,44 @@ interface HomePageState
 {
 }
 
+type TreeStruct = {label:string, children?:TreeStruct[]};
+
 export default class HomePage extends React.Component<HomePageProps, HomePageState>
 {
+	treeData:TreeStruct;
+
+	constructor(props:HomePageProps)
+	{
+		super(props);
+
+		// mock data for test
+		this.treeData = {label: "All", children: [
```

Comment:
Once the mockData PR is merged #52, I think we should try to follow that pattern for getting mockData. But fine to code inline during dev